src/ResourceManager/Sql/Commands.Sql/Commands.Sql.csproj

```diff
@@ -472,13 +471,25 @@
       <HintPath>..\..\..\packages\Microsoft.Data.Services.Client.5.8.2\lib\net40\Microsoft.Data.Services.Client.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
@jaredmoo you can remove the references added in this file (`Microsoft.Rest.ClientRuntime`, `Microsoft.Rest.ClientRuntime.Azure`, and `Newtonsoft.Json`) since they are already handled in the `Common.Dependencies.targets` file.